.github/workflows/publish.yaml

```diff
@@ -21,39 +27,33 @@ jobs:
       fail-fast: false
       matrix:
         node_version:
-          - 14
           - 16
           - 18
```

Comment:
Should add `20` at the same time if possible, it's going to be LTS-ified soon. To the other node version specifications too.